google/ggrc-core
# GGRC-8630 Replace can-map with can-define-map for MAPPERS, CLONERS components

# Issue description

Actually can-map, can-list are used. can-map, can-list are legacy and the alternatives in latest versions of CanJS are DefineMap, DefineList

# Steps to test the changes

Need to check:
 - Mappers
 - Global search
 - Assessment template clonner
 - Task bulk update
 - Assessment bulk verify

# Sanity checklist




# PR Review checklist
